lib/deps/ajax/ajaxCore.js

```diff
@@ -87,19 +86,7 @@ function ajax(options, callback) {
 
   return request(options, function (err, response, body) {
     if (err) {
-      // Node request only ever fires error or response, never both
```

Comment:
Well, technically that code path is still there, just as part of the ternary expression below. But it's not important enough, so no need to follow up on it. :)